lib/assertions.ts

```diff
@@ -575,16 +586,20 @@ export default class DOMAssertions {
    *
    * @see {@link #hasClass}
    */
-  hasPseudoElementStyle(selector: string | null, expected: object, message?: string): void {
+  hasPseudoElementStyle(
+    selector: string | null,
+    expected: { [key: string]: any },
```

Comment:
Can we define a more meaningful type for this? I think this is loosely a dictionary, correct?

```ts
interface Dictionary<T> {
  [key: string]: T;
}
```

Reply:
yes that is a good improvment